### PR TITLE
A few small changes

### PR DESCRIPTION
**Pull request by hlasimpk**
- Altered the probability score calculation following logistic regression work
- Added in a function to alter the space group of an mtz file so that if an alternative space group is selected by molrep, the mtz can be changed accordingly. 
- Added in .bat files for Charles as he wants to incorporate SIMBAD into windows.

### CODE REVIEW

File: simbad/util/mtz_util.py [hunk 1]
```diff
@@ -408,6 +410,46 @@ def crystal_data(mtz_file):
     return space_group, resolution, cell_parameters
 
 
+def change_space_group(input_mtz, output_mtz, new_space_group):
```
Comment (fsimkovic) [inline simbad/util/mtz_util.py]: I think this function should be incorporated in the class above. We have all the I/O stuff there anyway so a simple property should do. I would be much cleaner IMO.

Comment (hlasimpk) [reply simbad/util/mtz_util.py]: I've made these changes

File: simbad/util/mtz_util.py [hunk 2]
```diff
@@ -232,13 +269,13 @@ def create_reconstructed_amplitude_array(self, anomalous_amplitude_array):
...
 
-    def get_array_types(self, all_miller_arrays):
+    def get_array_types(self):
         """Function to assign array types contained within cctbx obj, in cases where there are multiple instances of
         a type of array, only the first will be considered. Due to limitations in cctbx
 
         Parameters
         ----------
-        all_miller_arrays : list
+        self.all_miller_arrays : list
```
Comment (fsimkovic) [inline simbad/util/mtz_util.py]: Don't need that here since this function doesn't take any parameters anymore

File: simbad/util/mtz_util.py [hunk 3]
```diff
@@ -105,6 +112,36 @@ def check_anomalous_arrays(miller_array):
             return True
         return False
 
+    def change_space_group(self, new_space_group):
+        """Change space group of input mtz
+
+        Parameters
+        ----------
+        input_mtz : str
```
Comment (fsimkovic) [inline simbad/util/mtz_util.py]: Should really be edited accordingly

File: simbad/util/mtz_util.py [hunk 4]
```diff
@@ -103,6 +112,36 @@ def check_anomalous_arrays(miller_array):
...
+            The path to the input mtz file
+        output_mtz : str
+            The path to the output mtz file
+        new_space_group : str
+            The new space group
+
+        Returns
```
Comment (fsimkovic) [inline simbad/util/mtz_util.py]: Function doesn't return anything either

File: simbad/version.py [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 # Maintain sematantic versioning. Further information can
 # be found here [http://semver.org/]
-__version_info__ = (0, 1, 9)
+__version_info__ = (0, 1, 11)
```
Comment (fsimkovic) [inline simbad/version.py]: Did we forget an update or why jump by 2?

Comment (hlasimpk) [reply simbad/version.py]: We forgot to update the version number to 0, 1, 10. Seeing as i'm going to make a release for this pretty soon I thought I might as well move it to 0, 1, 11

**Pull request merged**